Clients/GUI/Pages/InstrumentInfo.razor

```diff
@@ -20,7 +20,13 @@
             <ChartJsLineChart @ref="lineChartJs" Config="@lineConfig" Width="300" Height="150" />
         </div>
     </div>
+    
     <div class="col-md-6">
+        <div class="form-row input-group-prepend">
+            <label class="col-9 text-right">Current balance:  @currentBalance  @currency</label>
+            <input type="button" class="btn-link col-md-3" value="Increase balance" @onclick="GoToBalancePage">
```

Comment:
Make this also btn-primary I guess, so it will look better

Reply:
made btn-light. Primary attracts too much attention